src/rpc/calls/getCode.ts

```diff
@@ -0,0 +1,52 @@
+import { RPCError, RPCRequest, RPCResponse } from '../../types/types'
+import { callStarknet } from '../../utils/callHelper'
+import { validateEthAddress } from '../../utils/validations'
+import {
+  getSnAddressFromEthAddress,
+} from '../../utils/wrapper'
+
+export async function getCodeHandler(
+  request: RPCRequest,
+): Promise<RPCResponse | RPCError> {
+  if (request.params.length == 0) {
```

Comment:
```
if (request.params.length != 2)
```
If the function requires a specific number of parameters, just check for that ;)